data/completion/etelpmoc.sh

```diff
@@ -180,7 +180,7 @@ if [[ "${_comp[*]}" ]]; then
                 _compfunc="$OPTARG"
                 ;;
             W)
-                readarray -t COMPREPLY < <( builtin compgen -W "$OPTARG" -- "${COMP_WORDS[$COMP_CWORD]}" )
+                readarray -t COMPREPLY < <( builtin compgen -W "$OPTARG" -- "${COMP_WORDS[COMP_CWORD]}" )
```

Comment:
Is this related to the bug in question? It looks like a bug fix but I don't see how it relates to the 3rd argument.

Reply:
I believe it's just a run by fix for not needing `$i` for array indices as those are already treated like arithmetic evaluation.